src/test/unit_tests/test_nc_health.js

```diff
@@ -188,8 +188,10 @@ mocha.describe('nsfs nc health', function() {
             valid_system_json.config_directory = {
                 'config_dir_version': config_fs.config_dir_version,
                 'upgrade_package_version': pkg.version,
-                'phase': CONFIG_DIR_PHASES.CONFIG_DIR_UNLOCKED
+                'phase': CONFIG_DIR_PHASES.CONFIG_DIR_UNLOCKED,
```

Comment:
Why do you need the single quotes here? (I saw you removed them in the parts above)

Reply:
I don't, thanks